static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -94,13 +92,25 @@ function GroupRelatedIssues({params}: Props) {
               <HeaderWrapper>
                 <Title>{t('Issues caused by the same root cause')}</Title>
                 {sameRootCauseIssues.length > 0 ? (
-                  <GroupList
-                    endpointPath={`/organizations/${orgSlug}/issues/`}
-                    orgSlug={orgSlug}
-                    queryParams={{query: `issue.id:[${sameRootCauseIssues}]`}}
-                    query=""
-                    source="related-issues-tab"
-                  />
+                  <div>
+                    <TextButtonWrapper>
+                      <div />
+                      <LinkButton
+                        href={`/issues/?query=issue.id:[${groupId},${sameRootCauseIssues}]`}
```

Comment:
all links should still have the org path in them for non hybrid cloud setups
```suggestion
                        href={`/organizations/${orgSlug}/issues/?query=issue.id:[${groupId},${sameRootCauseIssues}]`}
```